### PR TITLE
update sig-cli leader

### PR DESCRIPTION
**Pull request by adohe**
@pwittrock I just update the docs, ptal.

### CODE REVIEW

**Comment by pwittrock** [comment]
SOrry for the late review.  Looks good.  Would you resolve the conflict?

**Comment by mengqiy** [comment]
@adohe Can you please rebase it?

File: sig-cli/README.md [hunk 1]
```diff
@@ -16,7 +16,6 @@ Covers kubectl and related tools. We focus on the development and standardizatio
   * [Meeting recordings](https://www.youtube.com/playlist?list=PL69nYSiGNLP28HaTzSlFe6RJVxpFmbUvF).
 
 ## Leads
-* Fabiano Franz (**[@fabianofranz](https://github.com/fabianofranz)**), Red Hat
```
[the remark this answers is not in the document]
Comment (cblecker) [reply sig-cli/README.md]: an example of how to do this: https://github.com/kubernetes/community/pull/1779

**Comment by mengqiy** [comment]
@adohe I guess now you are back to work after the holidays. Can you please resolve the merge conflict?

**Comment by mengqiy** [comment]
@adohe Ping

**Comment by adohe** [comment]
@mengqiy sure, I just go back from the Spring Festival Holiday, will update this today.

**Comment by adohe** [comment]
@pwittrock I just update this, ptal ~ :)

**Comment by mengqiy** [comment]
/lgtm
Thanks!

**Pull request merged**